### ISSUE TITLE
does it support batching ?

### ISSUE THREAD

**Issue by notshivansh** [kaltura/nginx-kafka-log-module#14]
Does kafka module use batching, while sending data. Moreover, the parameter  kafka_log_kafka_buffer_max_messages have any role in batching ?

**Comment by erankor** [comment]
Yes, this module uses librdkafka, message batching is implemented there. 
The parameter kafka_log_kafka_buffer_max_messages  is translated to librdkafka's `queue.buffering.max.messages`, which is the maximum number of messages in the output queue. By default, messages can wait in the queue up to 1 sec, before they are sent (`queue.buffering.max.ms`)
